### DISCUSSION TITLE
Quickstart: Svelte - Guide Broken

### DISCUSSION THREAD

**Discussion by rogadev** [supabase/supabase#6057] [General]
Hello! I followed the guide here: [https://supabase.com/docs/guides/with-svelte](https://supabase.com/docs/guides/with-svelte)

- I was unable to get a working app at the end of it. `use:someFunction` appears not to be a thing and I get intellisense yelling at me.
- In Profile.svelte line 78 or so `<input id="email" type="text" value={$user.email} disabled />` I get intellisense telling me that `$user.email` email is not a property, because, of course, we set it to `false` initially, and .email is not a property of false.
- The image in the Avatar section does not show and nothing I do to try to get it to render seems to work.

What am I missing?

**Comment by vinliao** [comment]
Not a direct answer to your question, but might be helpful: use SvelteKit with `svelte-add` instead. You can spin up a ready-to-go Svelte environment with Supabase integrated with two commands. More details: https://github.com/supabase-community/svelte-supabase.

**Comment by robots4life** [comment]
Hi @rogadev if you like to give SvelteKit and Supabase a try I am happy to take you through it step by step. You can find me in the Supabase Discord https://discord.supabase.com/, same nick, or just reply here and we will figure something out, we could peer code it if you like.

I have an example repo with a simple Todo app here https://github.com/robots4life/val, there is a local branch and a supabase branch, that could help you.

As for just Svelte and Supabase, I have not used Svelte in a long time, always SvelteKit since it has the router and all included, however, get in touch and we can just code it with SvelteKit if you like.
